Formula/aftman.rb

```diff
@@ -0,0 +1,30 @@
+class Aftman < Formula
+  desc "Toolchain manager for Roblox, the prodigal sequel to Foreman"
+  homepage "https://github.com/LPGhatguy/aftman"
+  url "https://github.com/LPGhatguy/aftman/archive/refs/tags/v0.2.7.tar.gz"
+  sha256 "2c4f191bfce631abff185a658b947105055da687c409e09ea80786be4c32b75e"
+  license "MIT"
+  head "https://github.com/LPGhatguy/aftman.git", branch: "main"
+
+  depends_on "rust" => :build
+
+  on_linux do
+    depends_on "pkg-config" => :build
+    depends_on "openssl@3"
+  end
```

Comment:
Are we sure this doesn't use OpenSSL on macOS?

Reply:
Output from `cargo tree` does not include the package `openssl` on macOS, whereas on Linux it does. Im assuming it has to do with builtin functionality with CoreFoundation. Both of the packages above are needed to compile the package `openssl-sys` on Linux.